### DISCUSSION TITLE
Unexpected rendering 'author field' in 'rawImports=true' mode

### DISCUSSION THREAD

**Discussion by unknownnumbers** [retorquere/zotero-better-bibtex#2889] [Q&A]
I want to import bibtex data with latex mode without any change. But it seems that in this mode('rawImports=true'), the author field is single field instead of double field. I want to set the default format of the author field to be double field as before. The reason why I want to maintain the format is to maintain the citekey format. A simple example and difference is as follows. What can I do? Any suggestions and hints will be appreciated.
![1](https://github.com/retorquere/zotero-better-bibtex/assets/28758103/657dff81-5887-42f3-a91e-8774be62c8b1)
![2](https://github.com/retorquere/zotero-better-bibtex/assets/28758103/dfb0e4e7-1b9c-4978-932b-3a44067676d4)

**Comment by retorquere** [comment]
Please open an issue for this
